packages/composites/dxdao/README.md

```diff
@@ -0,0 +1,82 @@
+# Chainlink External Adapter for Dxdao
+
+### Environment Variables
+
+| Required? |            Name            |               Description                |       Options       | Defaults to |
+| :-------: | :------------------------: | :--------------------------------------: | :-----------------: | :---------: |
+|    ✅     | `RPC_URL`  |   The RPC URL to connect to the XDai chain    |  |             |
+
+This adapter also relies on the Token Allocation adapter to fetch the USD price of ETH.  This requires setting up the environment variables 
+for the token allocation adapter as well.  The instructions for that can be found here https://github.com/smartcontractkit/external-adapters-js/tree/develop/packages/composites/token-allocation.
+
+
+---
+
+### Input Parameters
+
+No input parameters required
+
+---
+
+## Dxdao Endpoint
+
+This endpoint fetches the TVL(Total Value Locked) inside a pair that is deployed on the XDai chain.  The TVL is returned in USD.
+
+### Input Params
+
+| Required? |            Name            |               Description                |       Options       | Defaults to |
+| :-------: | :------------------------: | :--------------------------------------: | :-----------------: | :---------: |
+|    ✅     | `wethContractAddress`  |   The WETH contract address on the XDai Chain    |  |             |
```

Comment:
Do you think it would make sense to make this an env var?
WETH contract address should be dependent on which RPC_URL you have set (which blockchain you're connected to)

Reply:
I think having it as an env var is fine too.  I don't really have a preference on this.